src/node_mgmt/commands.py

```diff
@@ -179,11 +179,12 @@ def _set_address_holykell(mod: minimalmodbus.Instrument, result: dict,
         mod.address = original_slave_id
         try:
             mod.write_register(80, target_slave_id, 0, 6)
+            sleep(1)
         except minimalmodbus.IllegalRequestError:
```

Comment:
@nulcell could you check the flow of the code here. I'm 99.9% sure it should result in expected behaviour (as in first try the register 80, if the exception happens then try again with register 18, and if this will also results in an exception then the outer exception code block execute

Reply:
Yes, it would work that way.